frontend/public/locales/fr/translation.json

```diff
@@ -1966,10 +1966,10 @@
   "No status": "Aucun statut",
   "No users in group": "Aucun utilisateur dans le groupe",
   "No violations": "Aucune violation",
-  "no violations: {{count}} cluster": "aucune violation :{{count}} cluster",
-  "no violations: {{count}} cluster_plural": "aucune violation :{{count}} cluster",
-  "no violations: {{count}} policy": "aucune violation :{{count}} policy",
-  "no violations: {{count}} policy_plural": "aucune violation :{{count}} policy",
+  "no violations: {{count}} cluster": "aucune violation: {{count}} cluster",
+  "no violations: {{count}} cluster_plural": "aucune violation: {{count}} cluster",
+  "no violations: {{count}} policy": "aucune violation: {{count}} policy",
+  "no violations: {{count}} policy_plural": "aucune violation: {{count}} policy",
```

Comment:
Some colons had space before, space before and after, or no space at all. It was inconsistent, so I followed the English convention.